modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenResponse.java

```diff
@@ -27,6 +27,7 @@
     public boolean is3xx;
     public boolean is4xx;
     public boolean is5xx;
+    public String wildcardFirstChar; // store the first character of 2XX, 5XX, etc
```

Comment:
@wing328 I don't think this naming is intuitive (Users could assume this is firstChar of something other than status).

Maybe `statusCodeClass` or `statusCodeCategory`? The term "class" is taken directly from [RFC 7231](https://tools.ietf.org/html/rfc7231#section-6), but I'd be concerned this could become confused with the response class.

Reply:
I think rather than a mutable field, this can be a getter as it should always be the same based on the contents of this type.

Something like:

```
    public String getStatusCodeCategory() {
        if (code != null && code.length() > 0)
            return String.valueOf(code.charAt(0));
        return null;
    }
```